src/bp/ui-admin/yarn.lock

```diff
@@ -1451,9 +1451,9 @@
   resolved "https://registry.yarnpkg.com/@xtuc/long/-/long-4.2.1.tgz#5c85d662f76fa1d34575766c5dcd6615abcd30d8"
   integrity sha512-FZdkNBDqBRHKQ2MEbSC17xnPFOhZxeJ2YGSfr2BKf3sujG49Qe3bB+rGCwQfIaA7WHnGeGkSijX4FuBCdrzW/g==
 
-"@yaireo/tagify@botpress/tagify":
```

Comment:
can we discard that ?

Reply:
Pretty sure this is intentional but wasn't commit @MaxCloutier  ? 

I can remove this out of this PR but should be addressed as some of us might not have the proper deps.

Reply:
It needs to be there with the version, otherwise it causes weird bugs because it never gets updated on subsequent builds